osi_referenceline.proto

```diff
@@ -121,23 +123,23 @@ message ReferenceLine
     // coordinate.
     //
     // Example:
-    // \image html OSI_ReferenceLine1.svg
-    //
-    // This shows a reference line (consisting of three points), and five points
-    // not on the reference line.
-    //
-    // - For \c P1, the situation is clear, since there is exactly one nearest
-    //   point on the polyline. The resulting ST coordinate uniquely maps back
-    //   to \c P1.
-    // - \c P2 has multiple points "nearest points" on the polyline.
-    //   As can be seen here, two  ST coordinates map to \c P2 (red and grey
-    //   dotted line).  Following the rules above, the one with the smallest S
-    //   value is chosen (the red dotted line).
-    // - \c P3 has a unique "nearest point" on the polyline. However, multiple
-    //   points map to the same ST coordinate as that of \c P3, e.g. \c P4
-    //   (drawn in grey).
-    // - Finally, \c P5 shows how the reference line is extended infinitely for
-    //   points that are "outside" the reference line.
+    // \image html OSI_ReferenceLine1.svg "S, T calculation"
+    //
+    // This shows a reference line (consisting of three points R0, R1 and R2)
+    // and two points (P1 and P2) not part of the reference line.
+    //
+    // Calculation of ST for P1:
+    // - Calculate the instersection point I of the T axes of R0 and R1.
+    // - As P1 lies in the sector defined by these T axes it is considered part
```

Comment:
This change removes the \c formatting markers. I found these helpful for readability when reading the rendered document (as will be displayed on the website). I suggest you check how it looks in Doxygen, and probably re-add the \c formatting markers.

Reply:
Agree, these should be added again.